packages/react-preferences/src/components/PreferenceList.tsx

```diff
@@ -13,15 +13,92 @@ export const StyledList = styled.div`
   background: rgba(255, 255, 255, 0.2);
 `;
 
+const PreferenceSectionWrapper = styled.div`
+  background: white;
+  margin: 10px;
+  padding: 10px;
+  text: black;
+`;
+
+const SectionHeader = styled.h1`
+  margin: 0;
+  color: black;
+`;
+
+const LineBreak = styled.div`
+  height: 1px;
+  background-color: black;
+  widht: 100%;
+  opacity: 0.3;
+  margin: 8px 0;
+`;
+
+const PreferenceSection: React.FunctionComponent<{
+  section: any;
+  preferences: any;
+}> = ({ section, preferences }) => {
+  return (
+    <PreferenceSectionWrapper>
+      <SectionHeader>{section.name}</SectionHeader>
+      {section.preferenceGroups.nodes.map((template) => (
+        <>
+          <PreferenceTemplate
+            key={template.templateId}
+            preferenceTemplate={template}
+            recipientPreference={preferences?.recipientPreferences?.find(
+              (preference) => preference.templateId === template.templateId
+            )}
+            routingOptions={section.routingOptions}
+          />
+          <LineBreak />
+        </>
+      ))}
+    </PreferenceSectionWrapper>
+  );
+};
+
 export const PreferenceList: React.FunctionComponent<{
   // TODO: define Preferences theming
   theme?: ThemeProps<any>;
 }> = (props) => {
   const { brand } = useCourier();
   const preferences = usePreferences();
 
+  const renderPreferences = () => {
+    if (preferences?.isLoading) {
```

Comment:
do we want to be checking if sections are loading?

Reply:
This checks if everything getting loaded into preferences, including sections, is being loaded. So it covers that base.